src/lib/utils/definitions.ts

```diff
@@ -16,3 +16,6 @@ export const AFTER_ATTRIBUTE = '(?!\\w)';
 export const ANY_OPENING_TAG = '<.*>';
 export const ANY_CLOSING_TAG = '</.*>';
 export const ANY = '.*';
+export const ANY_ELEMENT_CHARACTER = '[^<>]';
```

Comment:
Disallow `<` and `>` in a Element, then It can be ignore that a regex  matches unexpected closing tag.